lib/index.ts

```diff
@@ -27,7 +36,9 @@ module.exports = (playlistURL, options) => {
   }
   let relativeBegin = typeof options.begin === 'string';
   let begin = relativeBegin ?
-    parseTime.humanStr(options.begin) :
+    // @ts-ignore this won't be called if options.begin isn't a string
```

Comment:
suppresses typescript errors, to use when they are invalid. in this case, typescript threw an error that `options.begin` could not be a string, but it's invalid since this only executes if `relativeBegin` is true, and it's only true if it's a string